src/network/networkd-wiphy.c

```diff
@@ -86,6 +87,137 @@ int wiphy_get_by_name(Manager *manager, const char *name, Wiphy **ret) {
         return 0;
 }
 
+static int link_get_wiphy(Link *link, Wiphy **ret) {
```

Comment:
I would move this function and `link_rfkilled()` to network-link and keep the rest here.

Reply:
networkd-link.c is too large, I'd like to place them at networkd-wiphy.c.